.deployment.config.json

```diff
@@ -271,7 +344,8 @@
         "extra_parameters": {
           "run-id": "$[GITHUB_RUN_ID]"
         }
-      }
+      },
+      "dependencies": ["smoke"]
     }
   ],
   "observatory": {
```

Comment:
Regarding the package_rollout, should it behave differently for the nightly? E.g., do we still want to deploy the nightly even if some PRs don't have an associated Jira?

Reply:
> Regarding the package_rollout, should it behave differently for the nightly? E.g., do we still want to deploy the nightly even if some PRs don't have an associated Jira?

Yeah: If that happens don't worry, I'll push red buttons till it passes